.github/workflows/Dev builds.yml

```diff
@@ -0,0 +1,117 @@
+name: Dev Builds
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+on:
```

Comment:
Add manual trigger?

Reply:
hmmm wat who needs such a thing??

Reply:
We do, lol.

Reply:
The docker.yml has a ` workflow_dispatch:` already.

Reply:
Now we just need it on docker_dev.yml

Reply:
done 👍

Reply:
@walkxcode